### PR TITLE
Add an extra line under Report Info to indicate if Full Report or Quick Tip

### LINKED ISSUE #2311: Difficulty Identifying FYI Analyst Reports Bug
It is currently difficult to identify FYI analyst reports due to default automated responses.

On the summary page of the site, the field options for the FYI reports are : What happened?, Supporting evidence, Your location, Your contact details (pictured bellow). In the FYI analyst reports, the list of fields also includes Incident info., Suspect Details and Financial Details. Since the user is never prompted to enter this information, the analyst reports automatically mention '' No Data'' in the ''Suspect Details'' and ''Financial Details'' fields. As for the Incident Information, the default response is '' i'm not sure ''. (pictures bellow)

The additional fields that are not meant to be part of the FYI flow make the identification of those reports difficult. Therefore, we should consider not only removing those fields (such as Incident info., Suspect Details and Financial Details), but also insert a header in the analyst reports to identify any FYI flow. 

### **Summary page fields** 
githubbb

### **FYI Analyst Reports additional fields (Photo 1)**
image001 (2)

### **FYI Analyst Reports additional fields (Photo 2)**
Screen Shot 2020-09-24 at 4 39 30 PM

### PR DESCRIPTION
**Pull request by DianeLiu2019**
Fixes #2311(issue)

# Description
currently, we don't the report type (full report or quick tip)  from the analyst report. The PR add an extra line under "Report Information  " section to indicate if it is Full Report or Quick Tip
> Please include a summary of the change.

# Any new packages installed?

> Give details

# Required followup work

> Is there anything related to this that still needs to be done (ex: documentation changes).

# Checklist:

- [ ] I have updated the azurescript.sh with any **new environment variables** and added them to the appsettings
- [x ] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [ ] I have added and needed tests for my changes (in particular for new screens)
- [ ] I have added a comment to any confusing code
- [ ] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

File: f2/src/utils/formatAnalystEmail.js [hunk 1]
```diff
@@ -60,6 +60,8 @@ const formatReportInfo = (data) => {
 
   let reportLanguage = data.language === 'fr' ? 'Français' : 'English'
 
+  let fyiForm = data.fyiForm === 'yes' ? 'Qick Tip' : 'Full Report'
```
Comment (justinr86) [inline f2/src/utils/formatAnalystEmail.js:63]: This is not bilingual.

File: f2/src/locales/en.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
   "analystReport.dateReceived": "Date received",
   "analystReport.fieldsMissing.warning": "Fields missing from above report",
   "analystReport.flagged": "Flagged",
+  "analystReport.fyiForm": "Flow Type",
```
Comment (justinr86) [inline f2/src/locales/en.json:19]: Just wondering about the wording of Flow Type. Will this be clear to the analysts? I think Report Type may be better.

Comment (DianeLiu2019) [reply f2/src/locales/en.json:19]: @stephaniemgauthier Can we get your opinion here , which one is better " Flow Type" or "Report Type" ?

**Comment by stephaniemgauthier** [comment]
I agree that "Report Type" will be better understood by the Analysts.

File: f2/src/utils/formatAnalystEmail.js (hunk 1 above)
Comment (DianeLiu2019) [reply f2/src/utils/formatAnalystEmail.js]: Fixed it

**Review by justinr86 — APPROVED** [review]

**Pull request merged**